app/views/foods/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<section>
+  <% if notice %>
+    <h4 style="color: green"> Food saved successfully</h4>
+  <% end %>
+  <button type="button" class="new-food-button align-near">
+    <%= link_to "Add food", new_food_path %>
+  </button>
+  <table id="foods" class="information-box">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Measurement unit</th>
+        <th>Unit Price</th>
+        <th>Action</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <%= @food.name %>
+        </td>
+        <td>
+          <%= @food.measurement_unit %>
```

Comment:
[OPTIONAL]
- I would suggest that this should be a dropdown menu having a list of the measurement units you intend the users to select.